src/main/java/Gab.java

```diff
@@ -0,0 +1,63 @@
+import java.util.Scanner;
+
+public class Gab {
+
+    private static Task[] taskList = new Task[100]; //array of Task instances
+
+    public static void getTask() {
+        String taskDescription;
+        Scanner in = new Scanner(System.in);
+        System.out.println("\tWhat do you want to do?: ");
+        taskDescription = in.nextLine();
+        checkKeywords(taskDescription); //check if its un mark/mark/exit/list
+    }
+
+    public static void checkKeywords (String task) { //going to input task description from getTask here
+        String[] taskAction = task.split(" ");
+        String action = taskAction[0];
```

Comment:
It would be good if you added some String Cleaning such as changing all to lower cases and trimming to make the input processing more robust.